src/components/InputTime/utils.test.tsx

```diff
@@ -381,7 +382,8 @@ describe('getDateTimeFromShortTimeString', () => {
         timeZoneName: 'America/Chicago',
       });
 
-      const dateIST = getDateTimeFromShortTimeString('12:30', {
+      const dateIST = getDateTimeFromShortTimeString({
+        time: '12:30',
         timeZoneOffset: +330,
       });
 
```

Comment:
some good edge cases to add tests for:
- offset brings you into the previous day
- offset brings you into the next day
- offset brings you into the next day during daylight savings time? :shudder:

Reply:
Curses. That is a very good idea.